services/ordering/src/EShopOnAbp.OrderingService.Application/Orders/OrderAppService.cs

```diff
@@ -200,11 +200,24 @@ private List<PaymentDto> CreatePaymentDtoMapping(List<Order> orders)
         return payments;
     }
 
+    private List<TopSellingDto> CreateTopSellingDtoMapping(List<Order> orders)
+    {
+        var orderItems = orders.Select(p => p.OrderItems).SelectMany(p => p).ToList();
+
+        var topSetlling = orderItems
+                .GroupBy(p => new { p.ProductName, p.PictureUrl })
```

Comment:
Product Id is the only field that cannot be changed, should we group by Id of product instead of string name?

Reply:
@gterdem What do you think about it?

Reply:
Agreed, we should use `ProductId` for grouping.